Sources/PostgresNIO/Connection/PostgresConnection+Configuration.swift

```diff
@@ -13,7 +13,7 @@ extension PostgresConnection {
             // MARK: Initializers
             
             /// Do not try to create a TLS connection to the server.
-            public static var disable: Self = .init(base: .disable)
+            public static let disable: Self = .init(base: .disable)
```

Comment:
Good catch!, But rather than making it a `let`, which also has concurrency implications due to lazy initialization, let's make it a stored property:

```suggestion
            public static var disable: Self { .init(base: .disable) }
```